### ISSUE TITLE
Aur package installs to subdir?

### ISSUE THREAD

**Issue by 2e0byo** [Anomalocaridid/handlr-regex#28]
I'm probably being daft here, but shouldn't

```bash
install -Dm 755 "handlr-${pkgver}" -t "${pkgdir}/usr/bin/handlr"
```

In the aur package be

```bash
install -Dm 755 "handlr-${pkgver}" "${pkgdir}/usr/bin/handlr"
```
?

Don't we want to install a binary named `handlr`, not a binary named `handlr-version` in a directory named `handlr`?

**Comment by Anomalocaridid** [comment]
No, you're absolutely right. The `handlr-regex-bin` AUR package was installing everything in the wrong places. It should be fixed now.

Thanks for catching it!
